### PR TITLE
Create fr.yml

### PR DESCRIPTION
**Pull request by KazeroG**
### Pull Request summary

translate bazaar to FR !

### Checklist

- [ ] New code is covered by tests
- [ ] Extension installation works
- [x] Extension update works
- [ ] Extension uninstallation works

### CODE REVIEW

**Comment by KazeroG** [comment]
error not related to translation !

<details>```
$ composer global require hirak/prestissimo
Changed current directory to /home/travis/.composer
Using version ^0.3.7 for hirak/prestissimo
./composer.json has been created
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
  - Installing hirak/prestissimo (0.3.7): Downloading (100%)
Writing lock file
Generating autoload files
5.30s$ composer install --prefer-dist -n
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.
  Problem 1
    - The requested package flarum/core ^0.1.0-beta.8 is satisfiable by flarum/core[0.1.x-dev] but these conflict with your requirements or minimum-stability.
  Problem 2
    - The requested package flagrow/console ^0.4 exists as flagrow/console[0.1.0, 0.2, 0.3, dev-b8, dev-master] but these are rejected by your constraint.
  Problem 3
    - Installation request for flagrow/steamroller dev-master -> satisfiable by flagrow/steamroller[dev-master].
    - flagrow/steamroller dev-master requires flarum/core 0.1.0-beta.7 -> satisfiable by flarum/core[v0.1.0-beta.7] but these conflict with your requirements or minimum-stability.
```</details>

File: resources/locale/fr.yml [hunk 1]
```diff
@@ -0,0 +1,122 @@
+flagrow-bazaar:
+    admin:
+        loader:
+            is_loading: Traitement ...
+            error: ne erreur est survenue. Consultez l'onglet Historique et votre journal Flarum pour plus de détails. N'ayez pas peur de nous le signaler
```
Comment (clarkwinkelmann) [inline resources/locale/fr.yml]: Manque un bout de mot

File: resources/locale/fr.yml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                title: Flagrow Bazaar
+                field:
+                    token:
+                        label: API token
+                        description: Ce jeton d'api est requis pour accéder à l'extension de Flagrow. Si déjà défini, vous êtes prêt à partir!
+                    use_cron_for_tasks:
+                        label: Cronjobs
+                        description: >
+                          En permettant à cronjobs de traiter les tâches Bazaar, ces processus
+                           sont moins susceptibles d'expirer. S'il vous plaît noter que vous devez être en mesure de
```
Comment (clarkwinkelmann) [inline resources/locale/fr.yml]: Mauvais alignement

File: resources/locale/fr.yml [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+                    description: >
+                        Nous vous recommandons d'autoriser PHP à utiliser au moins {required} de la mémoire, votre php.ini autorise actuellement {limit}.
+                        Bien que nous essayions d'augmenter la limite de mémoire pendant l'exécution, le succès dépend de votre environnement d'hébergement.
+                        Pour plus d'informations, reportez-vous à <a>le wiki</a>.
+        page:
+            state:
+                no_results_available: >
+                    Il semble qu'aucun résultat n'était disponible pour le moment.
+                not_authorized: >
+                    otre autorisation contre notre flagrow.io api a échoué. Peut-être une connexion Internet instable est à l'origine de cela.
```
Comment (clarkwinkelmann) [inline resources/locale/fr.yml]: Manque un bout de mot

**Comment by clarkwinkelmann** [comment]
@KazeroG let's stay on topic. Please report that error to whoever is concerned, and it does not look like a Bazaar issue.

I've noted a few issues with your translations. Also please note we plan to do further work on Bazaar before the next release so I'm not sure whether we will merge this before or after the changes. Thanks for your contribution(s) :smile:

**Comment by luceos** [comment]
@clarkwinkelmann the error is caused by our master branch, it already required flarum beta 8, without telling composer how to load that beta 8 you won't be able to install dependencies for Bazaar on the current dev-master 👍

**Comment by clarkwinkelmann** [comment]
I'm closing this PR as no further changes have been made and because #132 will be validated very soon. If you still have suggestions for the french translation, please submit a new PR to alter the existing french translation once it's merged. Thanks!

**Pull request closed without merging**